### PR TITLE
readme: fix minor punctuation errors

### PR DESCRIPTION
**Pull request by joonkyu**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
- Added/removes punctuation to make sure it's consistency throughout the README file.

#suricata-verify-pr:
#suricata-verify-repo:
#suricata-verify-branch:
#suricata-update-pr:
#suricata-update-repo:
#suricata-update-branch:
#libhtp-pr:
#libhtp-repo:
#libhtp-branch:

### CODE REVIEW

**Comment by jufajardini** [comment]
Hello joonkyu, thanks for your contribution, did you really mean to close this one?

**Comment by joonkyu** [comment]
I did not mean to close it! I was trying to clear out my profile, may have deleted the fork by accident.

File: README.md [hunk 1]
```diff
@@ -100,14 +100,14 @@ If the PR was closed without comments it's likely due to QA failure. If the Gith
 
 __Q: the compiler/code analyser/tool is wrong, what now?__
```
Comment (jufajardini) [inline README.md:101]: Could you change the initial letter to capital, like you did elsewhere, too?

**Review by jufajardini — CHANGES_REQUESTED** [review]
Hello, overall, the changes look good. I've asked for one addition, that goes along other changes you've already done.

We also need to fix the commit message. Our commit style indicates that we put the module/file we've edited, and then summarize what the commit does, like so:

`readme: fix minor punctuation errors`

Thanks for your interest in contributing to Suricata! :)

File: README.md (hunk 1 above)
Comment (joonkyu) [reply README.md:101]: Unfortunately I have deleted the fork so I'm not sure how I'm supposed to edit the file now. Is there any way to do so?

Comment (joonkyu) [reply README.md:101]: Okay, I'll have to remake then, how do I link the new changes to this pull request, do I have to make a new pull request as well?

Comment (jufajardini) [reply README.md:101]: In general, we prefer that changes are incorporated into new pull requests, so this would the way, yes :)

Comment (joonkyu) [reply README.md:101]: Great, I'll close this pull request and make a new one with the appropriate changes.

**Pull request closed without merging**